app/javascript/components/cloud-object-store-container-form/cloud-object-store-container-form.schema.js

```diff
@@ -0,0 +1,38 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+// eslint-disable-next-line max-len
+const url = '/api/providers?expand=resources&attributes=id,name,supports_cloud_object_store_container_create&filter[]=supports_cloud_object_store_container_create=true';
+const loadStorageManagers = () => API.get(url).then(({ resources }) => {
+  let storageManagerOptions = [];
+  storageManagerOptions = resources.map(({ id, name }) => ({ label: name, value: id }));
+  storageManagerOptions.unshift({ label: `<${__('Choose')}>`, value: '-1' });
```

Comment:
Any particular reason why you are keeping manual placeholder instead of using  `includeEmpty: true`?

Reply:
The include empty does not have a value so it does not cause the on change function to run. This means we can't use the emptyschema function.